wire-ios-request-strategy/Sources/E2EIdentity/ACME/AcmeClient.swift

```diff
@@ -21,6 +21,8 @@ import WireCoreCrypto
 
 public protocol AcmeClientInterface {
     func getACMEDirectory() async throws -> Data
+    func getACMENonce(path: String) async throws -> String
```

Comment:
It seems annoying and error prone to pass these paths around, the mapping is clear and they all come from the AcmeDirectory.

Wouldn't it make sense to inject the `AcmeDirectory` into the  the `AcmeClient` and then `path` argument could be removed? The call call to load the AcmeDirectory would then need to live somewhere else,  `AcmeDirectoryAPI`?

Reply:
If you don't want to do that now you could at least move the `AcmeDirectory` inside the `E2eIEnrollment` so the methods it exposes doesn't need to have any path arguments.

Reply:
Moved `AcmeDirectory` to the `E2eIEnrollment`